MultimodalQnA/tests/test_compose_on_gaudi.sh

```diff
@@ -20,7 +20,7 @@ export caption_fn="apple.txt"
 
 function build_docker_images() {
     cd $WORKPATH/docker_image_build
-    git clone https://github.com/opea-project/GenAIComps.git && cd GenAIComps && git checkout "${opea_branch:-"main"}" && cd ../
+    git clone https://github.com/mhbuehler/GenAIComps.git && cd GenAIComps && git checkout "${opea_branch:-"dina/image_query"}" && cd ../
```

Comment:
This is a temporary change and it would need to be changed to `mmqna-image-query` when my branch merges.